api/base.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+import "google/protobuf/timestamp.proto";
+option go_package = "internal/proto/base";
+
+package base;
+
+message DebugData {
+  google.protobuf.Timestamp ts = 1; // timestamp
+  string uuid = 2; // uuidv4
```

Comment:
can we call this just `id` -- it can be any arbitrary string IMHO

Reply:
Ok, the tradeoff for the flexibility would be that we drop the uniqueness assurances for the identifier, as well as no uniformity when it comes to the format of the identifier. Is this a tradeoff we are willing to make? if yes, what for?

Reply:
we don't care -- this is for the client to choose / set

Reply:
why do we not care? do we not like clean, predictable data?

Reply:
What I mean to say is: we should not make any assumptions about the request identifier -- the client may use whatever format they see fit -- we should not be prescriptive here IMHO

Reply:
okidoki

Reply:
1 sec

Reply:
renamed uuid to id in fbc31f9